pcsx2/GS/Renderers/Metal/convert.metal

```diff
@@ -547,7 +547,7 @@ fragment float4 ps_shadeboost(float4 p [[position]], DirectReadTextureIn<float>
 	float3 satColor = mix(intensity, brtColor, sat);
 	float3 conColor = mix(AvgLumin, satColor, con);
 
-	float3 csb = pow(conColor, float3(1.0 / gam));
+	float3 csb = pow(max(conColor, 0.0), float3(1.0 / gam));
```

Comment:
How could this be negative?

Reply:
No idea, only dx was complaining, I just changed it on all for consistency.

Reply:
Not a huge fan of compensating for problems we don't think we have.  Doing this means things like UBSan won't detect if we actually do have a situation where negative values end up in here.

Put a `#pragma warning (disable : 3571)` in the hlsl shader